<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="headLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="phoneLabel">
           <property name="text">
            <string>null</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="genderLabel">
           <property name="text">
            <string>null</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>null</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnWork">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>工作</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_5">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>日记</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_6">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>规划</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pushButton_7">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>小说</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>备忘录</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_9">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>知识点</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_10">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>账户管理</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_11">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>汽车保养</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>工具类</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_12">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>项目管理</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_14">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>API管理</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton_16">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>数据库</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>爬虫</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_13">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>音视频</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_15">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>二维码</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pushButton_17">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>文件解析</string>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="minimumSize">
      <size>
       <width>630</width>
       <height>80</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>668</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSend">
        <property name="minimumSize">
         <size>
          <width>84</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

      border: 2px solid #8f8f91;
      border-radius: 6px;
	 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  }
  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                        stop: 0 #dadbde, stop: 1 #f6f7fa);
  }
  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }
  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
